projects/canopy/src/lib/spacing/spacing.interface.ts

```diff
@@ -10,3 +10,11 @@ export type SpacingVariant =
   | 'xxl'
   | 'xxxl'
   | 'xxxxl';
+
+export interface ResponsiveSpacing {
+  sm?: SpacingVariant;
+  md?: SpacingVariant;
+  lg?: SpacingVariant;
+  xlg?: SpacingVariant;
```

Comment:
Shouldn't these keys be `xl` / `xxl` to follow the naming conventions?

Reply:
good call, it could probably use these?
https://github.com/Legal-and-General/canopy/blob/master/projects/canopy/src/lib/shared/breakpoints.interface.ts